src/NServiceBus.AcceptanceTests/Core/DelayedDelivery/TimeoutManager/When_dispatch_fails_on_removal_SendsAtomicWithReceive.cs

```diff
@@ -11,7 +11,7 @@
     using NUnit.Framework;
     using Timeout.Core;
 
-    class When_timeout_dispatch_fails_on_timeout_removal_using_sendsAtomicWithReceive : NServiceBusAcceptanceTest
+    class When_dispatch_fails_on_removal_SendsAtomicWithReceive : NServiceBusAcceptanceTest
```

Comment:
This change doesn't seem to be related to the MSMQ permissions. Was that done purposely?